app/src/main/java/com/example/habithelper/activities/MainActivity.java

```diff
@@ -12,19 +12,29 @@
 import com.example.habithelper.fragments.MoodFragment;
 import com.example.habithelper.fragments.ProfileFragment;
 import com.example.habithelper.fragments.TrackFragment;
+import com.example.habithelper.models.Habit;
 import com.google.android.material.bottomnavigation.BottomNavigationView;
+import com.parse.FindCallback;
+import com.parse.ParseException;
+import com.parse.ParseQuery;
+import com.parse.ParseUser;
+
+import java.util.List;
 
 public class MainActivity extends AppCompatActivity {
 
     public static MainActivity self;
     final FragmentManager fragmentManager = getSupportFragmentManager();
     protected BottomNavigationView bottomNavigation;
+    public ParseUser currentUser;
+    List<String> badges;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         self = this;
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+        currentUser = ParseUser.getCurrentUser();
```

Comment:
Since the current user is persisted between sessions, the only way this is possible is if I delete a User manually from the Parse Database website and reload the app, so it won't be a problem for users. (I can put a null check in place if you would like).